satpy/modifiers/angles.py

```diff
@@ -33,18 +33,21 @@
 from pyorbital.astronomy import cos_zen as pyob_cos_zen
 from pyorbital.astronomy import get_alt_az
 from pyorbital.orbital import get_observer_look
-from pyresample.geometry import AreaDefinition, SwathDefinition
+from pyresample.geometry import AreaDefinition, StackedAreaDefinition, SwathDefinition
 
 import satpy
 from satpy.utils import get_satpos, ignore_invalid_float_warnings
 
-PRGeometry = Union[SwathDefinition, AreaDefinition]
+PRGeometry = Union[SwathDefinition, AreaDefinition, StackedAreaDefinition]
 
 # Arbitrary time used when computing sensor angles that is passed to
 # pyorbital's get_observer_look function.
 # The difference is on the order of 1e-10 at most as time changes so we force
 # it to a single time for easier caching. It is *only* used if caching.
 STATIC_EARTH_INERTIAL_DATETIME = datetime(2000, 1, 1, 12, 0, 0)
+DEFAULT_UNCACHE_TYPES = (SwathDefinition, xr.DataArray, da.Array)
+DEFAULT_UNHASH_TYPES = DEFAULT_UNCACHE_TYPES
```

Comment:
Do you expect these two variables to diverge at some point? Otherwise do we really need them both?

Reply:
I made them two variables because I thought you were going to disagree with me using a "uncacheable" variable and an "unhashable" variable for the same thing lol.

Reply:
Done.